### PR TITLE
Make API installable

### PR DESCRIPTION
**Pull request by xecgr**
I need this api installable via pip
To do it, I've needed to move the very-important file InstagramAPI.py to InstagramAPI/__init__.py and I've also moved ImageUtils.py to this new folder    

I've tested and it seems to work fine
```
xecgr@carteras:~/dev$ virtualenv ve_test
New python executable in ve_test/bin/python
Installing setuptools, pip, wheel...done.
xecgr@carteras:~/dev$ cd ve_test/
xecgr@carteras:~/dev/ve_test$ source bin/activate
(ve_test)xecgr@carteras:~/dev/ve_test$ pip install -e git+https://github.com/xecgr/Instagram-API-python.git#egg=InstagramAPI
Obtaining InstagramAPI from git+https://github.com/xecgr/Instagram-API-python.git#egg=InstagramAPI
  Cloning https://github.com/xecgr/Instagram-API-python.git to ./src/instagramapi
Collecting requests==2.11.1 (from InstagramAPI)
/home/xecgr/dev/ve_test/local/lib/python2.7/site-packages/pip/_vendor/requests/packages/urllib3/util/ssl_.py:90: InsecurePlatformWarning: A true SSLContext object is not available. This prevents urllib3 from configuring SSL appropriately and may cause certain SSL connections to fail. For more information, see https://urllib3.readthedocs.org/en/latest/security.html#insecureplatformwarning.
  InsecurePlatformWarning
  Using cached requests-2.11.1-py2.py3-none-any.whl
Collecting requests-toolbelt==0.7.0 (from InstagramAPI)
  Using cached requests_toolbelt-0.7.0-py2.py3-none-any.whl
Collecting moviepy==0.2.3.2 (from InstagramAPI)
  Using cached moviepy-0.2.3.2-py2.py3-none-any.whl
Collecting decorator==4.0.11 (from moviepy==0.2.3.2->InstagramAPI)
  Using cached decorator-4.0.11-py2.py3-none-any.whl
Collecting tqdm==4.11.2 (from moviepy==0.2.3.2->InstagramAPI)
  Using cached tqdm-4.11.2-py2.py3-none-any.whl
Collecting imageio==2.1.2 (from moviepy==0.2.3.2->InstagramAPI)
Collecting numpy (from moviepy==0.2.3.2->InstagramAPI)
Collecting pillow (from imageio==2.1.2->moviepy==0.2.3.2->InstagramAPI)
Collecting olefile (from pillow->imageio==2.1.2->moviepy==0.2.3.2->InstagramAPI)
Installing collected packages: requests, requests-toolbelt, decorator, tqdm, olefile, pillow, numpy, imageio, moviepy, InstagramAPI
  Running setup.py develop for InstagramAPI
Successfully installed InstagramAPI decorator-4.0.11 imageio-2.1.2 moviepy-0.2.3.2 numpy-1.12.1 olefile-0.44 pillow-4.1.1 requests-2.11.1 requests-toolbelt-0.7.0 tqdm-4.11.2
/home/xecgr/dev/ve_test/local/lib/python2.7/site-packages/pip/_vendor/requests/packages/urllib3/util/ssl_.py:90: InsecurePlatformWarning: A true SSLContext object is not available. This prevents urllib3 from configuring SSL appropriately and may cause certain SSL connections to fail. For more information, see https://urllib3.readthedocs.org/en/latest/security.html#insecureplatformwarning.
  InsecurePlatformWarning
You are using pip version 7.1.2, however version 9.0.1 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
(ve_test)xecgr@carteras:~/dev/ve_test$ python
Python 2.7.6 (default, Jun 22 2015, 17:58:13) 
[GCC 4.8.2] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> from InstagramAPI import InstagramAPI
>>> 
>>> username = ''
>>> pwd      = ''
>>> API = InstagramAPI(username,pwd)
>>> API.login()
Request return 429 error!
Login success!

True
>>> 
(ve_test)xecgr@carteras:~/dev/ve_test$
```
 I've seen that there's already a pull request for that: https://github.com/LevPasha/Instagram-API-python/pull/43 
If you agree with this one, you'll be able to close the other    
I've also changed the readme to allow new users to install using the easy way

### CODE REVIEW

**Comment by mattccrampton** [comment]
+1 on this, it would be nice to just pull in via pip

**Comment by LevPasha** [comment]
@xecgr hide your IG login-password. What we need to do to make 'pip install InstagramApi'?

**Comment by xecgr** [comment]
As you can see, the credentials are fake: 
```
>>> username = 'my_username'
>>> pwd      = 'my_pwd'
```
I would like to install via pip to use it in heroku free instance where you can't access to machine (so you can't download, copy, paste, edit, etc files) but you can define the `requirements.txt` file that pip will install in the instance

**Comment by LevPasha** [comment]
@xecgr remove your login-password anyway

**Comment by xecgr** [comment]
Already removed!

File: examples/get_all_comments.py [hunk 1]
```diff
@@ -2,8 +2,8 @@
 import time
 from datetime import datetime
 
-username = ''
-pwd      = ''
+username = 'xiscugarcia'
+pwd      = 'xisquet'
 media_id = '1477006830906870775_19343908'
```
Comment (LevPasha) [inline examples/get_all_comments.py]: I talk about this

File: examples/user_followings.py [hunk 2]
```diff
@@ -2,21 +2,24 @@
 import time
 from datetime import datetime
 
-username = ''
-pwd      = ''
-user_id  = '19343908'
+username = 'xiscugarcia'
+pwd      = 'xisquet'
+username = 'Bowmacntmj5617'
+pwd = 'Zzrk9qzlrR'
```
Comment (LevPasha) [inline examples/user_followings.py]: This

File: examples/user_followings.py [hunk 3]
```diff
@@ -2,21 +2,24 @@
...
 
 
 API = InstagramAPI(username,pwd)
 API.login()
-
+user_id  = '200361318'
+API.getUsernameInfo(user_id)
+API.LastJson
 following   = []
```
Comment (LevPasha) [inline examples/user_followings.py]: This

File: test.py [hunk 4]
```diff
@@ -5,7 +5,7 @@
 
 from InstagramAPI import InstagramAPI
 
-InstagramAPI = InstagramAPI("login", "password")
+InstagramAPI = InstagramAPI("xiscugarcia", "xisquet")
 InstagramAPI.login() # login
```
Comment (LevPasha) [inline test.py]: This

**Comment by xecgr** [comment]
Sorry... I've just removed. Could you remove the comments?

File: README.md [hunk 5]
```diff
@@ -13,28 +13,15 @@ It is still a work in progress to copy all of its API endpoints.
...
-
-    `pip install -r requirements.txt`
-
-
-4. Modify test.py with your own username and password
-
-
-5. Run the test script (**use text editor to edit the script and type in valid Instagram username/password**)
-
-    `python test.py`
```
Comment (LevPasha) [inline README.md]: Can you stay all 2 method, buy pip and old one.

File: examples/user_followings.py [hunk 6]
```diff
@@ -3,20 +3,21 @@
...
 next_max_id = True
 while next_max_id:
     print next_max_id
     #first iteration hack
     if next_max_id == True: next_max_id=''
-    _ = API.getUserFollowings(user_id,maxid=next_max_id)
+    _ = API.getUserFollowers(user_id,maxid=next_max_id)
```
Comment (LevPasha) [inline examples/user_followings.py]: followings.py so getUserFollowings

File: examples/user_followings.py [hunk 7]
```diff
@@ -3,20 +3,21 @@
...
 username = ''
-pwd      = ''
-user_id  = '19343908'
+pwd = ''
 
 
 API = InstagramAPI(username,pwd)
 API.login()
-
+user_id  = ''
```
Comment (LevPasha) [inline examples/user_followings.py]: Move it to 7 line

File: test.py [hunk 8]
```diff
@@ -5,7 +5,7 @@
 
 from InstagramAPI import InstagramAPI
 
-InstagramAPI = InstagramAPI("login", "password")
+InstagramAPI = InstagramAPI()
```
Comment (LevPasha) [inline test.py]: InstagramAPI = InstagramAPI("login", "password"

**Comment by LevPasha** [comment]
Plz fix it, thx!

**Comment by xecgr** [comment]
Have a look

**Pull request merged**